app/gui/view/component-browser/searcher-list-panel/src/lib.rs

```diff
@@ -797,17 +874,23 @@ impl component::Frp<Model> for Frp {
         let selection_corners_animation = Animation::<f32>::new(network);
         let spring = inertia::Spring::default() * SELECTION_ANIMATION_SPRING_FORCE_MULTIPLIER;
         selection_animation.set_spring.emit(spring);
-        fn selection_position(model: &Model, id: GroupId, group_local_pos: Vector2) -> Vector2 {
+        fn selection_position(
+            model: &Model,
+            id: GroupId,
+            group_local_pos: Vector2,
+            style: &Style,
+        ) -> Vector2 {
             let scroll_area = &model.scroll_area;
-            let scroll_area_pos = scroll_area.position() + scroll_area.content().position();
+            let scroll_area_size = style.scroll_area_size();
+            let scroll_area_center = Vector2(-scroll_area_size.x / 2.0, scroll_area_size.y / 2.0);
+            let scroll_area_pos = scroll_area_center + scroll_area.content().position().xy();
             let section_pos = match id.section {
                 SectionId::Favorites => model.favourites_section.content.position(),
                 SectionId::LocalScope => default(),
                 SectionId::SubModules => model.sub_modules_section.content.position(),
             };
             let group_pos = model.groups_wrapper.get(&id).map(|g| g.position()).unwrap_or_default();
-            let pos = (scroll_area_pos + section_pos + group_pos).xy() + group_local_pos;
-            model.clamp_y(pos)
+            scroll_area_pos + (section_pos + group_pos).xy() + group_local_pos
         }
```

Comment:
I know it was not introdcuced in this PR, but can we make this function outside the function body?

Reply:
Done.